examples/contour/02-role-contour.yaml

```diff
@@ -21,29 +21,8 @@ rules:
   - ""
   resources:
   - endpoints
-  verbs:
```

Comment:
this change was just rearranging things to group them by API group, shouldn't be any actual changes to the rules.